### ISSUE TITLE
too many values to unpack (expected 2)

### ISSUE THREAD

**Issue by LazariPetru** [plesk/centos2alma#154]
Hi,

I tries to run the conversion by following the instructions, but unfortunately as soon as I run the command:  "./centos2alma --start"
Is showing : "Do preparation checks.." and after it stops and exit from the operation.

I checked the logs and I'm receiving the following error: 
2024-01-02 01:51:36,450 - ERROR - too many values to unpack (expected 2)
2024-01-02 01:51:36,455 - ERROR - Please fix noted problems before proceed the conversion

Can you please investigate why this error is happening?

Many thanks,
Appreciate your effort! Best Regards.

[centos2alma_feedback.zip](https://github.com/plesk/centos2alma/files/13809219/centos2alma_feedback.zip)

**Comment by SandakovMM** [comment]
Please run the command "centos2alma" again with the verbose mode enabled:
```shell
> ./centos2alma --verbose
```
Then, please share the logfile containing the results at: /var/log/plesk/centos2alma.log

**Comment by LazariPetru** [comment]
Is already in the zip: [centos2alma_feedback.zip](https://github.com/plesk/centos2alma/files/13809219/centos2alma_feedback.zip)

However, here is the log file:
[centos2alma.log](https://github.com/plesk/centos2alma/files/13809437/centos2alma.log)


Many Thanks

**Comment by SandakovMM** [comment]
> However, here is the log file:

I meant generate the same file, but in verbose mode =)
```shell
> ./centos2alma --verbose
```

**Comment by LazariPetru** [comment]
Sorry for my ignorance.

Here is the output: 

2024-01-02 12:49:02,138 - DEBUG - Start checks
2024-01-02 12:49:02,138 - DEBUG - Make check checking if distro is CentOS7
2024-01-02 12:49:02,139 - DEBUG - Make check checking if Plesk version is actual
2024-01-02 12:49:02,171 - DEBUG - Make check checking if Plesk installer is in progress
2024-01-02 12:49:02,192 - DEBUG - Make check checking available space
2024-01-02 12:49:02,193 - DEBUG - Make check checking outdated PHP
2024-01-02 12:49:02,585 - DEBUG - Make check checking if grub is installed
2024-01-02 12:49:02,586 - DEBUG - Make check checking if there is more than one NIC interface using ketnel-name
2024-01-02 12:49:02,586 - DEBUG - Make check checking if the system not in a container
2024-01-02 12:49:02,586 - DEBUG - Make check checking if the last installed kernel is in use
2024-01-02 12:49:07,678 - DEBUG - Installed kernel versions: 3.10.0-1160.102.1.el7.x86_64, 3.10.0-1160.95.1.el7.x86_64, 3.10.0-1160.92.1.el7.x86_64, 3.10.0-1160.90.1.el7.x86_64, 3.10.0-1160.99.1.el7.x86_64
2024-01-02 12:49:07,680 - ERROR - too many values to unpack (expected 2)
2024-01-02 12:49:07,680 - ERROR - Please fix noted problems before proceed the conversion

-----------------
Hm, I guess is because there are multiple kernels on the machine?   

Is there an alternative to make the conversion even so?

Many thanks

**Comment by SandakovMM** [comment]
Generally, having multiple versions of the kernel installed on the target instance should be okay, as long as you are running the latest version. It seems that there may be an issue with parsing information about the kernels.
Could you please provide the output of the following commands?
```shell
> uname -r
> rpm -q -a kernel
> rpm -q -a kernel-rt
```

**Comment by LazariPetru** [comment]
```
# uname -r
3.10.0-1160.90.1.vz7.200.7

# rpm -q -a kernel
kernel-3.10.0-1160.102.1.el7.x86_64
kernel-3.10.0-1160.95.1.el7.x86_64
kernel-3.10.0-1160.92.1.el7.x86_64
kernel-3.10.0-1160.90.1.el7.x86_64
kernel-3.10.0-1160.99.1.el7.x86_64

# rpm -q -a kernel-rt
(nothing) existing from the 

I tired this:
# rpm -qa | grep kernel
kernel-3.10.0-1160.102.1.el7.x86_64
kernel-3.10.0-1160.95.1.el7.x86_64
kernel-3.10.0-1160.92.1.el7.x86_64
kernel-3.10.0-1160.90.1.el7.x86_64
kernel-3.10.0-1160.99.1.el7.x86_64
kernel-devel-3.10.0-1160.92.1.el7.x86_64
kernel-tools-3.10.0-1160.102.1.el7.x86_64
kernel-devel-3.10.0-1160.90.1.el7.x86_64
kernel-devel-3.10.0-1160.102.1.el7.x86_64
kernel-devel-3.10.0-1160.99.1.el7.x86_64
kernel-tools-libs-3.10.0-1160.102.1.el7.x86_64
kernel-headers-3.10.0-1160.102.1.el7.x86_64
kernel-devel-3.10.0-1160.95.1.el7.x86_64
abrt-addon-kerneloops-2.1.11-60.el7.centos.x86_64
```

**Comment by SandakovMM** [comment]
I see, the issue is caused by the information from the currently installed version: "3.10.0-1160.90.1.vz7.200.7". I will look into what can be done about it.

**Comment by SandakovMM** [comment]
The problem should be fixed by #156. Please find attached archive from [related build](https://github.com/plesk/centos2alma/actions/runs/7386162539). 
[centos2alma.zip](https://github.com/plesk/centos2alma/files/13810270/centos2alma.zip)

**Comment by SandakovMM** [comment]
Release with changes has been [published](https://github.com/plesk/centos2alma/releases/tag/v1.2.2)
